src/influx_udp_worker.erl

```diff
@@ -60,11 +54,17 @@ init([{options, #{ host := Host, port := Port} = _Options}]) ->
 handle_call({write, Bin}, _From, State) when is_binary(Bin) ->
   {reply, send_data(State, Bin), State};
 
+handle_call({write, Points}, _From, State) ->
+  {reply, send_data(State, influx_line:encode(Points)), State};
+
 handle_call({write, Series, Points}, _From, State) ->
-  {reply, send_data(State, encode(Series, Points, [])), State};
+  {reply, send_data(State, influx_line:encode(Series, Points)), State};
 
 handle_call({write, Series, Points, Tags}, _From, State) ->
-  {reply, send_data(State, encode(Series, Points, Tags)), State};
+  {reply, send_data(State, influx_line:encode(Series, Points, Tags)), State};
+
+handle_call({write, Series, Points, Tags, Time}, _From, State) ->
+  {reply, send_data(State, influx_line:encode(Series, Points, Tags, Time)), State};
```

Comment:
Line 67 is too long:   {reply, send_data(State, influx_line:encode(Series, Points, Tags, Time)), State};.